packages/salesforcedx-vscode-replay-debugger/src/states/variableAssignmentState.ts

```diff
@@ -34,13 +37,38 @@ export class VariableAssignmentState implements DebugLogState {
         ref = this.fields[5];
       }
       let container: ApexVariableContainer | undefined;
+      let map: Map<String, VariableContainer> | undefined;
       if (logContext.getStaticVariablesClassMap().has(className)) {
-        const statics = logContext.getStaticVariablesClassMap().get(className)!;
-        container = statics.get(name)! as ApexVariableContainer;
+        map = logContext.getStaticVariablesClassMap().get(className)!;
```

Comment:
We're calling logContext.getStaticVariablesClassMap().get(className) twice, here and in the if statement above. Can't we just set map = logContext.getStaticVariablesClassMap().get(className) and check to see if map is defined?

Reply:
Yeah I thought this was a little easier to read and verify that the container and map were always tied together at the time but that makes sense too, I'll update it thanks